apps/web/src/components/organisms/profileMenu/menus/userMenu/UserMenu.tsx

```diff
@@ -16,7 +16,7 @@ export default function UserMenu() {
   return (
     <div className={styles.menu}>
       <div className={menuStyles.profile}>
-        <Avatar size={64} src={profilePicture} />
+        <Avatar size={64} src={!!user.avatar ? user.avatar : profilePicture} />
```

Comment:
```suggestion
        <Avatar size={64} src={user.avatar} />
```

komponent avatar obsługuje defaultowy avatar